Shopify/View/HomeScreens/HomeViewController.swift

```diff
@@ -8,6 +8,8 @@
 import UIKit
 
 class HomeViewController: UIViewController {
+    var homeViewModel : HomeViewModel?
```

Comment:
we have discussion that before, we should't have (View Model ) for every (View Controller,) however we must have view model for every functionality 
if U did that you have to change the name of the View Model class to what it did and be more specific in Your names